### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by CristianGM**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# RxPager
+RxPager is an Android library that helps handling paginated results in a reactive way
+
+Is based on [this link](https://gist.github.com/mttkay/24881a0ce986f6ec4b4d) from [@mttkay](https://gist.github.com/mttkay)
```
Comment (ferranpons) [inline README.md]: instead of "this link" maybe you should say "this gist"

File: README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+To get the content, just subscribe to: `pager.getPageObservable()`
+It will call onNext with the first page, and will continue giving to you the next pages when you call `pager.next()` and `complete` when the returned page have `null`as nextPageToken
+
+To know if its loading, there is another observable available: `pager.getIsLoadingObservable()`
+and last, there is `pager.hasNext()` wich returns a Boolean.
+
+##Common data types
+There is available a POJO called: TokenPage
+`TokenPage(String nextPageToken, List<ITEM> results)`
+being ITEM the tipe of your elements in the list.
```
Comment (ferranpons) [inline README.md]: typo

**Comment by ferranpons** [comment]
Maybe for a better comprehension you should provide an example

**Comment by CristianGM** [comment]
@ferranpons better?

**Comment by rocboronat** [comment]
Maybe adding the common gradle compile could be great. BTW, don't hesitate to merge this PR and go on in another PR :·)

**Comment by CristianGM** [comment]
@rocboronat Sure!
but first it needs to be published 😝

**Pull request merged**